mono/metadata/sre.c

```diff
@@ -3654,52 +3666,6 @@ ves_icall_TypeBuilder_create_runtime_class (MonoReflectionTypeBuilder *tb)
 	return NULL;
 }
 
-static gboolean
-reflection_initialize_generic_parameter (MonoReflectionGenericParam *gparam, MonoError *error)
-{
-	MonoImage *image;
-
-	mono_error_init (error);
-
-	image = &gparam->tbuilder->module->dynamic_image->image;
-
-	if (gparam->mbuilder) {
-		if (!gparam->mbuilder->generic_container) {
-			MonoType *tb = mono_reflection_type_get_handle ((MonoReflectionType*)gparam->mbuilder->type, error);
-			return_val_if_nok (error, FALSE);
-
-			MonoClass *klass = mono_class_from_mono_type (tb);
-			gparam->mbuilder->generic_container = (MonoGenericContainer *)mono_image_alloc0 (klass->image, sizeof (MonoGenericContainer));
-			gparam->mbuilder->generic_container->is_method = TRUE;
-			/* 
-			 * Cannot set owner.method, since the MonoMethod is not created yet.
-			 * Set the image field instead, so type_in_image () works.
-			 */
-			gparam->mbuilder->generic_container->is_anonymous = TRUE;
-			gparam->mbuilder->generic_container->owner.image = klass->image;
-		}
-	} else if (gparam->tbuilder) {
-		if (!gparam->tbuilder->generic_container) {
-			MonoType *tb = mono_reflection_type_get_handle ((MonoReflectionType*)gparam->tbuilder, error);
-			return_val_if_nok (error, FALSE);
-			MonoClass *klass = mono_class_from_mono_type (tb);
-			gparam->tbuilder->generic_container = (MonoGenericContainer *)mono_image_alloc0 (klass->image, sizeof (MonoGenericContainer));
-			gparam->tbuilder->generic_container->owner.klass = klass;
-		}
-	}
-
-	return TRUE;
-}
-
-void
-ves_icall_GenericTypeParameterBuilder_initialize (MonoReflectionGenericParam *gparam)
```

Comment:
There's another instance of this function left inside the DISABLE_REFLECTION_EMIT ifdef.